packages/frontend/src/server/features/scaling/activity/get-scaling-activity-entries.ts

```diff
@@ -81,16 +88,21 @@ export async function getScalingActivityEntries() {
   }
 }
 
-export type ScalingActivityEntry = SetOptional<
-  ReturnType<typeof getScalingProjectActivityEntry>,
-  'href'
->
+export interface ScalingActivityEntry extends CommonScalingEntry {
+  entryType: 'activity'
+  dataSource: ScalingProjectDisplay['activityDataSource']
+  dataAvailability: {
+    layer: DataAvailabilityWithSentiment['layer'] | undefined
+  }
+  data: ActivityProjectTableData
+}
+
```

Comment:
the pattern that we decided to use is to determine type based on function return value rather than typing the function, it is done like that everywhere and worked pretty smoothly so I would not change it